pkg/statistics/handle/bootstrap.go

```diff
@@ -181,7 +182,11 @@ func (h *Handle) initStatsHistograms4Chunk(is infoschema.InfoSchema, cache stats
 				zap.Stack("stack"))
 		}
 	}()
-	var table *statistics.Table
+	var (
+		table        *statistics.Table
+		tblInfo      tableinfo.Table
```

Comment:
now that it's cached, we can directly cache the `*TableInfo` instead of the `*Table` to avoid the calling of `Meta()`.